### PR TITLE
PA_PHPM-55 implement flyway

### PR DESCRIPTION
**Pull request by anup-7span**
implement flyway

### CODE REVIEW

File: friendlist/src/main/resources/application.yml [hunk 1]
```diff
@@ -12,14 +12,19 @@ spring:
   datasource:
     url: jdbc:postgresql://0.0.0.0:5432/friendlist_management
     username: postgres
-    password: root
+    password: test
```
Comment (hetvi-7span) [inline friendlist/src/main/resources/application.yml:15]: Do no add local password in commit

File: friendlist/src/main/resources/db/migration/V1__Create_Table.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CREATE
+    TABLE
+        friendlist_management.test(
+            id serial NOT NULL,
+            name VARCHAR(255),
+            uid VARCHAR(255),
+            PRIMARY KEY(id)
+        );
```
Comment (hetvi-7span) [inline friendlist/src/main/resources/db/migration/V1__Create_Table.sql]: Add actual table ddl

**Review by hetvi-7span — CHANGES_REQUESTED** [review]

File: friendlist/src/main/resources/db/migration/V1__Create_Table.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CREATE
```
Comment (hetvi-7span) [inline friendlist/src/main/resources/db/migration/V1__Create_Table.sql]: can we define a proper naming convention for file name ??

**Review by hetvi-7span — CHANGES_REQUESTED** [review]

File: friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
+-- public.friend_request definition
```
Comment (akshay-7span) [inline friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql]: Can we add a query to if schema exists or not?
CREATE SCHEMA [IF NOT EXISTS] schema_name;

File: friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    updated_by varchar(255) NULL,
+    message varchar(255) NULL,
+    receiver_id int4 NOT NULL,
+    sender_id int4 NOT NULL,
+    status varchar(255) NULL,
+    CONSTRAINT friend_request_pkey PRIMARY KEY (id)
+);
+-- public.friends definition
+
+-- Drop table
```
Comment (akshay-7span) [inline friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql]: what are these drop table comments?

**Review by akshay-7span — CHANGES_REQUESTED** [review]

File: friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql (hunk 4 above)
Comment (anup-7span) [reply friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql]: wee added schema in the configuration file file

so do need to extra query?

**Review by hetvi-7span — APPROVED** [review]

**Review by akshay-7span — APPROVED** [review]

**Pull request merged**